consumer.py

```diff
@@ -1,29 +1,38 @@
-from robocorp.tasks import task
+import requests
 from robocorp import workitems
+from robocorp.tasks import task
 
-import requests
 
 @task
 def consume_traffic_data():
     """
-        Inhuman Insurance, Inc. Artificial Intelligence System robot.
-        Consumes traffic data work items.
+    Inhuman Insurance, Inc. Artificial Intelligence System robot.
+    Consumes traffic data work items.
     """
     process_traffic_data()
 
+
 def process_traffic_data():
     for item in workitems.inputs:
         payload = item.payload
         traffic_data = payload["traffic_data"]
         valid = validate_traffic_data(traffic_data)
-        if(valid):
+        if valid:
             status, return_json = post_traffc_data_to_sales_system(traffic_data)
             if status == 200:
                 item.done
-            else: 
-                item.fail(exception_type="APPLICATION", code="TRAFFIC_DATA_POST_FAILED", message=return_json["message"])
+            else:
+                item.fail(
+                    exception_type="APPLICATION",
+                    code="TRAFFIC_DATA_POST_FAILED",
+                    message=return_json["message"],
+                )
         else:
-            item.fail(exception_type="BUSINESS", code="INVALID_TRAFFIC_DATA", message=item.payload)
+            item.fail(
+                exception_type="BUSINESS",
+                code="INVALID_TRAFFIC_DATA",
+                message=item.payload,
+            )
 
 
 def validate_traffic_data(traffic_data):
```

Comment:
Not visible in the diff, but in general instead of `if condition: return True else: return False` you can just write `return condition`. Maybe this makes more sense in the context of the course, it's up to you.

Reply:
I am not even sure. There weren't many ifs until now. It may be simpler if it is explicit? 
But then again, I kinda hope that they already have a python basic at this point.